wiki/Twitter/vi.md

```diff
@@ -0,0 +1,24 @@
+# Twitter
+
+osu! có một vài tài khoản Twitter khác nhau để phục vụ những mục đích khác nhau. Phần lớn tài khoản này không hoạt động nhiều nhưng chúng sẽ giúp bạn cập nhật với tin tức về osu!.
+
+## Dịch vụ
+
+| Ảnh đại diện | Tên tài khoản | Mô tả |
+| :-: | :-: | :-- |
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Nơi thông báo về những trục trặc của website và [Bancho](/wiki/Bancho_(server)). |
+| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Hỗ trợ bạn về tài khoản và/hoắc những vấn đề cộng đồng. Được hoạt động bởi [đội hỗ trợ tài khoản](/wiki/People/The_Team/Account_support_team). |
```

Comment:
```suggestion
| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Hỗ trợ bạn về tài khoản và/hoặc những vấn đề cộng đồng. Được vận hành bởi [đội hỗ trợ tài khoản](/wiki/People/The_Team/Account_support_team). |
```
`run` here means `to maintain`, `to operate`, so you should use `điều hành` or `vận hành`. Or in some cases it means `to manage`, at which you use `quản lý`.